Need help in building image, missing fsl_common.h

# ⚠️ Fails at 80% of make: could not find fsl_common.h 

## 📝 Description
To make `./configure linux RelWithDebInfo` pass I had to install some -devel packages, namely libgtkmm-3-devel and fuse2-devel (I'm running SolusOS, a rolling release Linux distro). With this and previously installed -devel packages the configure stage finished successfully, but when I got to 80% of `make` in the build folder, i get this fatal error:

`~/Mudita/MuditaOS/third-party/usb_stack/device/usb.h:14:10: fatal error: fsl_common.h: could not be found`

Any ideas how to continue the build?

## 📝 Describe the solution you'd like
Figure out what package needs to be installed in my OS or in Ubuntu to continue the build.

## 📝 Describe alternatives you've considered
Building in Docker instead, haven't tried yet.

--- a/products/BellHybrid/CMakeLists.txt
+++ b/products/BellHybrid/CMakeLists.txt
@@ -1,6 +1,6 @@
 add_executable(BellHybrid)
 
-add_custom_target(Bell
+add_custom_target(Bell ALL
     DEPENDS BellHybrid BellHybrid.img
     )
 

--- a/products/PurePhone/CMakeLists.txt
+++ b/products/PurePhone/CMakeLists.txt
@@ -1,6 +1,6 @@
 add_executable(PurePhone)
 
-add_custom_target(Pure
+add_custom_target(Pure ALL
     DEPENDS PurePhone PurePhone.img
     )
 

